src/app/(employee)/getting-started/_components/ResumeUpload.tsx

```diff
@@ -0,0 +1,91 @@
+"use client"
+import React, { useState } from 'react';
+import Button from '@/components/Button';
+import { IMAGES } from '@/assets';
+import axios from 'axios';
+import Image from 'next/image';
+import { toast } from 'sonner';
+
+// Define types for the props
+interface ResumeUploadProps {
+  setSelectedFile: (file: File | null) => void;
+  onUploadSuccess: () => void;
+}
+
+const ResumeUpload: React.FC<ResumeUploadProps> = ({ setSelectedFile, onUploadSuccess }) => {
+  const [selectedFile, setLocalSelectedFile] = useState<File | null>(null);
+
+  const handleFileChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const file = event.target.files?.[0] || null;
+    setLocalSelectedFile(file);
+    setSelectedFile(file); // Update the parent state
+  };
+
+  const handleFileClick = () => {
+    const fileUpload = document.getElementById('file-upload') as HTMLInputElement;
+    if (fileUpload) {
+      fileUpload.click();
+    }
+  };
+
+  const handleUpload = async () => {
+    if (selectedFile) {
+      try {
+        const fileData = new FormData();
+        fileData.append('file', selectedFile); // Use 'file' as the field name
+
+        await axios.put('https://carrerhub-backend.vercel.app/api/v1/resumes', fileData, {
```

Comment:
donot harcode the url and function like this, extract to a promise and use it with tanstack/react-query